### ISSUE TITLE
[Bug]: multiple accessibility problems

### ISSUE THREAD

**Issue by mehgcap** [debba/tabularis#86]
### Describe the bug

Tabularis has some accessibility problems for screen reader users. For the following, I used 0.9.6 on Windows 11 with the NVDA screen reader.

The biggest problem is the query editor.

1.  NVDA does not read any text entered into the field, making it impossible to review or edit a query. I had to use a different editor and then paste my query.
2. If I tried to type, I was jumped to a list of keyword suggestions. This caused NVDA to exit focus mode (the mode where the user can type instead of having letters interpreted as screen reader-specific navigation commands). This made for a jarring and frustrating experience. Of course, I quickly found that I couldn't review anything in the query field anyway.
3. I had to press control-enter to submit the query. That's fine, and I like when that command is used. The problem is that it was not stated anywhere, and there was no button to submit the query.

Other things I noticed:

* In the table of query results, every table cell is spoken as clickable. Normally, this indicates that an element that isn't normally actionable (td, span, p, and so on) will do something when clicked or when a screen reader user presses enter or space on it. Doing so on these cells, even the header cells, did nothing at all. Should it have sorted the results or something?
* When adding a connection, it would be nice if the test results were spoken using a live region, or gained focus automatically.
* In the list of all tables in the database, there is an unlabeled button after each table name that seems to open more details about it. This button needs a label, and ARIA should be used to indicate when a table's information is expanded or collapsed.

### To Reproduce

To reproduce, just use the application with NVDA on Windows. I suspect other platforms will run into similar problems.

### OS Version

Windows 11

### Tabularis Version

0.9.6

### Relevant Log Output

```shell

```

**Comment by arturbent0** [comment]
Hi, i would like to work on improving the issues described here if thats alright @debba

**Comment by debba** [comment]
Of course, feel free to work on it, I will assign the task to you, thanks @arturbent0
